packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -225,6 +225,19 @@ const createEntityPageListItems = (
     )}/${entityId}`;
   };
 
+  const { accountId, universe } = parseYextrcContents();
+  // Content is knowledge graph
+  const formatContentLink = (uid: string) => {
+    if (accountId !== undefined && universe !== undefined) {
+      const partition = getPartition(Number(accountId));
```

Comment:
Move this outside of the function so it only needs to execute once. I'm not sure if it'll just return empty string or throw error if accountId is undefined, so you'll have to try it out.

Reply:
just empty string

Reply:
By _this_ I was referring to moving the getPartition call outside of the formatContentLink function.

Reply:
Oh I see, oops